stylesheets/v2-classic-dark.css

```
@@ -36,6 +36,7 @@ body {
 
 .v-Atom {
   background: #212733;
+  color: #d9d9d9;
```

```suggestion
  color: inherit;
```

Before I merge can you change this to a `color: inherit` so that there's less duplication and less to think about in the future.

`color` is [deeply inheritable](https://developer.mozilla.org/en-US/docs/Web/CSS/inherit) so this will reach all the way back up to the earliest available `color` and set it properly.